### DISCUSSION TITLE
[Implemented] Specification for `plotIndividualTimeProfile()`

### DISCUSSION THREAD

**Discussion by IndrajeetPatil** [Open-Systems-Pharmacology/OSPSuite-R#815] [Ideas]
Related to #674 

# Signature

```r
# `...` will be parameters to control annotations and aesthetics of the plot
plotIndividualTimeProfile(dataCombined, ...)
```

# Implementation plan

```r
data <- dataCombined$toDataFrame()

tlf::plotTimeProfile(data, ...) 
```

# Rough expected output:

![image](https://user-images.githubusercontent.com/11330453/157679090-7e92fef9-a2e7-4790-b925-9acbaa6e962c.png)

# Decision points:

N.B. The option already rejected by the team is scratched out.

# Tasks

Each task will get its own PR.

- Implement `plotIndividualTimeProfile()`

**Comment by IndrajeetPatil** [comment]
@PavelBal Both individual and population profile plots should work across all the following contexts, right? 

- `DataCombined` has _only_ observed data. 
- `DataCombined` has _only_ simulated data. 
- `DataCombined` has _both_ observed and simulated data.

My understanding is yes, but just want to have this information included in the specification as well.

**Comment by PavelBal** [comment]
Yes, sure.

**Comment by IndrajeetPatil** [comment]
`plotIndividualTimeProfile` and `plotPopulationTimeProfile` line type should be solid.
